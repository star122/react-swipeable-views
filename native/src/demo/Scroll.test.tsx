import 'react-native';
import * as React from 'react';
import Scroll from './Scroll';
import * as renderer from 'react-test-renderer';

describe('Scroll snapshot', () => {
  jest.useFakeTimers();

  it('renders the root', async () => {
    const tree = renderer.create(<Scroll />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
