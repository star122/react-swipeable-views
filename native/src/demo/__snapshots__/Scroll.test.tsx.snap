// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll snapshot renders the root 1`] = `
<View
  animateTransitions={true}
  index={0}
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
        "overflow": "hidden",
      },
      undefined,
    ]
  }
/>
`;
